src/LazyLoadImage/LazyLoadImage.stories.tsx

```diff
@@ -108,6 +110,70 @@ const Template: StoryFn<LazyLoadImageProps> = args => {
   );
 };
 
+const FlexSizeComponent: StoryFn<LazyLoadImageProps> = args => {
+  return (
+    <Grid
+      key={"Example project code"}
+      sx={theme => ({
+        "&:active": {
+          border: `2px solid ${theme.palette.primary.main}`
+        },
+        "&:hover": {
+          borderColor: theme.palette.primary.main
+        },
+        "&:hover > div > h5": {
+          color: theme.palette.primary.main,
+          transition: "color 0.1s"
+        },
+        backgroundColor: theme.palette.background.paper,
+        border: `1px solid`,
+        borderRadius: "6px",
+        cursor: "pointer",
+        height: "238px",
+        transition: "border-color 0.3s"
+      })}
+      container
+    >
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "column",
+          px: 3,
+          py: 2,
+          width: "100%"
+        }}
+      >
+        <NoWrapTypography
+          variant="h5"
+          sx={{
+            fontWeight: 700
+          }}
```

Comment:
Please wrap this in a box to avoid the visible jumbing in the story, same should applied in same use cases in VIRTO apps.